### DISCUSSION TITLE
Scroll on application change app's volume (new feature)

### DISCUSSION THREAD

**Discussion by ericktucto** [linux-is-awesome/gnome_extension_rocketbar#2] [Ideas]
Thanks for Rocketbar.
Please, can you include in the extension to change the volume of the application when scrolling the icon?

**Comment by ChepKun** [comment]
Thanks for the idea. Added to the board https://github.com/orgs/linux-is-awesome/projects/1

**Comment by ChepKun** [comment]
@ErickTucto hey!
This feature has been merged to the develop branch along with other new features. I'm going to test everything for a while before creating a public release (a week or so). In case you would like to try it right now and probably give some feedback, please follow the Manual installation steps from the readme to install the Unstable version.
This feature has some limitations and I'm not really sure if it fits your personal needs, so your feedback would be helpful!
